bioptim/interfaces/biomodel.py

```diff
@@ -192,3 +192,39 @@ def contact_forces(self, q, qdot, tau, external_forces: list = None) -> MX:
 
     def passive_joint_torque(self, q, qdot) -> MX:
         """Get the passive joint torque"""
+
+    def q_range(self) -> list:
+        """Get the q_ranges"""
+
+    def qdot_range(self) -> list:
+        """Get the qdot_ranges"""
+
+    def qddot_range(self) -> list:
+        """Get the qddot_ranges"""
+
+    def q_and_q_dot_range(self, index_to_keep):
```

Comment:
This method does not make much sense in an API point of view. We already provide q and qdot ranges, why would we need to supply q_and_qdot too?

Reply:
Outdated